src/drivers/mct/shr/seq_comm_mct.F90

```diff
@@ -199,14 +199,15 @@ integer function seq_comm_get_ncomps()
   end function seq_comm_get_ncomps
 
 
-  subroutine seq_comm_init(Comm_in, nmlfile)
+  subroutine seq_comm_init(Comm_in, nmlfile, Comm_ID)
 
     !----------------------------------------------------------
     !
     ! Arguments
     implicit none
     integer, intent(in) :: Comm_in
     character(len=*), intent(IN) :: nmlfile
+    integer, optional, intent(in) :: Comm_ID
```

Comment:
Could you please add a sentence or two as to why we need the Comm_ID  optional argument. We need to explain how this new argument differentiates between multiple couplers and 1 component instance per coupler versus a single coupler and multi-component instances per that one coupler.